content/en/cloud_cost_management/container_cost_allocation.md

```diff
@@ -24,7 +24,7 @@ Clouds
 Resources
 : CCM allocates costs for Kubernetes clusters and includes cost analysis for many associated resources such as Kubernetes persistent volumes used by your pods.
 
-CCM displays costs for resources including CPU, memory, and more depending on the cloud and orchestrator you are using on the [**Containers** page][1].
+CCM displays costs for resources including CPU, memory, and GPU and more depending on the cloud and orchestrator you are using on the [**Containers** page][1].
```

Comment:
This has 2 `and`s

I would vote to just not mention GPUs here and the `and more` should cover that